be/src/exec/parquet/column_reader.h

```diff
@@ -27,6 +30,268 @@ struct ColumnReaderOptions {
     std::string timezone;
 };
 
+// When doing decimal convert, source scale may not equal with destination scale,
+// when destination scale is greater than source, source unscaled data will be scaled up
+// to match destination scale.
+enum DecimalScaleType {
+    NO_SCALE,
+    SCALE_UP,
+    SCALE_DOWN,
+};
+
+template <typename T>
+struct DecimalTypeTraits {
+    using PrimitiveType = T;
+};
+
+template <>
+struct DecimalTypeTraits<DecimalV2Value> {
+    using PrimitiveType = int128_t;
```

Comment:
`PrimitiveType` already exists, such as `TYPE_INT`, `TYPE_VARCHAR`.

Reply:
removed and revised